src/python/ddapp/ikconstraintencoder.py

```diff
@@ -15,14 +15,16 @@ class ConstraintEncoder(nje.NumpyEncoder):
     def default(self, obj):
         if isinstance(obj, vtk.vtkTransform):
             pos, quat = transformUtils.poseFromTransform(obj)
-            return OrderedDict(position=pos, quaternion=quat)
+            return OrderedDict(position=pos.tolist(), quaternion=quat.tolist())
         elif isinstance(obj, fieldcontainer.FieldContainer):
             d = OrderedDict()
             d['class'] = type(obj).__name__
             for key in obj._fields:
                 d[key] = getattr(obj, key)
             return d
-        return nje.NumpyEncoder.default(self, obj)
+        # return nje.NumpyEncoder.default(self, obj)
+        # Return Numpy array as list
+        return obj.tolist()
```

Comment:
do we want to make this change?  It assumes the object is a numpy data type.  Maybe in this context that is a correct assumption, not sure.